packages/Webkul/Admin/src/Listeners/Customer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Webkul\Admin\Listeners;
+
+use Illuminate\Support\Facades\Mail;
+use Webkul\Admin\Mail\Customer\RegistrationNotification;
+
+class Customer extends Base
+{
+    /**
+     * After customer is created
+     *
+     * @param  \Webkul\Customer\Contracts\Customer  $customer
+     * @return void
+     */
+    public function afterCreated($customer)
+    {
+        try {
+            if (! core()->getConfigData('emails.general.notifications.emails.general.notifications.customer_registration_confirmation_mail_to_admin')) {
+                return;
+            }
+
+            Mail::queue(new RegistrationNotification($customer));
```

Comment:
@amit-webkul,
No, we cannot use ($this->prepareMail) instead of Mail::queue here. Mail::queue and ($this->prepareMail) serve different purposes.

(Mail::queue) is used to send emails asynchronously, where the email is sent in the background, and you don't have to wait for a response on the site.

On the other hand, ($this->prepareMail) is a method associated with mailable classes. It is used to customize the aspects of the email before it is sent.

Reply:
**@sunarcrajneesh ,**

Here ($this->prepareMail) is a global method for mailing. Inside this method, we used Mail::queue.Which works with locale and currency. You can use this method for doing the same.